controllers/comments.controllers.js

```diff
@@ -0,0 +1,14 @@
+const { selectAllCommentsByArticleId } = require("../models/comments.models");
+
+exports.getAllCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { sort_by, order } = req.query;
```

Comment:
In this ticket you haven't been asked to take `sort_by` or `order` queries, returned comments should just be sorted by `created_at`
It a good extra feature, but outside the scope of this ticket 🔍